src/Blocks/components/checkbox/components/checkbox-options.js

```diff
@@ -39,119 +39,117 @@ export const CheckboxOptions = (attributes) => {
 	const checkboxIsHidden = checkAttr('checkboxIsHidden', attributes, manifest);
 
 	return (
-		<>
-			<PanelBody title={__('Checkbox', 'eightshift-forms')}>
-				<Section icon={icons.options} label={__('General', 'eightshift-forms')}>
-					<NameField
-						value={checkboxValue}
-						attribute={getAttrKey('checkboxValue', attributes, manifest)}
-						disabledOptions={checkboxDisabledOptions}
-						setAttributes={setAttributes}
-						label={__('Value', 'eightshift-forms')}
-						type={'checkbox'}
-						isChanged={isNameChanged}
-						setIsChanged={setIsNameChanged}
-					/>
-				</Section>
+		<PanelBody title={__('Checkbox', 'eightshift-forms')}>
+			<Section icon={icons.options} label={__('General', 'eightshift-forms')}>
+				<NameField
+					value={checkboxValue}
+					attribute={getAttrKey('checkboxValue', attributes, manifest)}
+					disabledOptions={checkboxDisabledOptions}
+					setAttributes={setAttributes}
+					label={__('Value', 'eightshift-forms')}
+					type={'checkbox'}
+					isChanged={isNameChanged}
+					setIsChanged={setIsNameChanged}
+				/>
+			</Section>
 
-				<Section icon={icons.tag} label={__('Label', 'eightshift-forms')}>
-					<IconToggle
-						label={__('Use label', 'eightshift-forms')}
-						checked={!checkboxHideLabelText}
-						onChange={(value) => setAttributes({ [getAttrKey('checkboxHideLabelText', attributes, manifest)]: !value })}
-						reducedBottomSpacing
-					/>
+			<Section icon={icons.tag} label={__('Label', 'eightshift-forms')}>
+				<IconToggle
```

Comment:
Why not wrap the section in a UseToggle so you don't have to do this? 😄

Reply:
we agreed on slack that we have issue in multiple versions of libs so I will leave this as is